hail/python/hail/backend/service_backend.py

```diff
@@ -21,6 +22,13 @@ def __init__(self, deploy_config=None):
         self.url = deploy_config.base_url('query')
         self.headers = service_auth_headers(deploy_config, 'query')
         self._fs = None
+        self._logger = None
+
+    @property
+    def logger(self):
+        if self._logger is None:
+            self._logger = PythonOnlyLogger()
+        return self._logger
```

Comment:
Sorry we keep going back and forth on this.  I think:
 - you should pass skip_logging_configuration from init(_service) to the backend (ask the Spark backend does)
 - the logger construction shouldn't be lazy

Then you don't need a dummy configure on the Spark backend which is misleading because the configuration happens elsewhere.

Reply:
Good point. Will fix.